cmd/kubectl_token.go

```diff
@@ -402,15 +408,17 @@ func basicAuth(input *LoginInput) (managementClient.Token, error) {
 			"[%v] message:[%v]", apiError["code"], apiError["message"])
 	}
 
-	err = json.Unmarshal(response, &token)
+	err = json.Unmarshal(respBody, &token)
 	if err != nil {
 		return token, err
 	}
+
 	return token, nil
 }
 
-func samlAuth(input *LoginInput, client *http.Client) (managementClient.Token, error) {
+func samlAuth(client *http.Client, input *LoginInput) (managementClient.Token, error) {
```

Comment:
The idiomatic approach is to list arguments in the order of increasing specificity. This way all dependencies come first.